freqtrade/rpc/external_message_consumer.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         self.enabled = self._emc_config.get('enabled', False)
         self.producers: List[Producer] = self._emc_config.get('producers', [])
 
-        self.wait_timeout = self._emc_config.get('wait_timeout', 300)  # in seconds
+        self.wait_timeout = self._emc_config.get('wait_timeout', 30)  # in seconds
```

Comment:
> The wait_timeout has been updated to 30 seconds instead of 300 to better support reverse proxies closing connections if no data is sent after a certain time.

I'm not sure about this point. To me, this means that the reverseproxy should be reconfigured.

It'll increase our ping frequency from 1/5m to 2/1m - which is quite a lot more - so i think 30 is too low.